<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shape2DCircleDialog</class>
 <widget class="QDialog" name="Shape2DCircleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_x">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_y">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>radius</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_radius">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="btnB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
